docs/source/notes/autograd.rst

```diff
@@ -170,6 +170,32 @@ processed by autograd internally: default mode (grad mode), no-grad mode,
 and inference mode, all of which can be togglable via context managers and
 decorators.
 
+
+.. list-table::
+   :widths: 50 50 50 50 50
+   :header-rows: 1
+
+   * - Mode
+     - required-grad takes effect
```

Comment:
nit: 
```suggestion
     - ``requires_grad`` takes effect
```

Reply:
autograd respects the ``requires_grad`` of operator inputs

Reply:
changed